libs/SmartStore/SmartStore/Classes/SmartStoreSDKManager.m

```diff
@@ -97,7 +97,7 @@ -(NSArray*) getDevActions:(UIViewController *)presentedViewController
 
 - (NSArray*) getDevSupportInfos
 {
-    SFSmartStore *store = [SFSmartStore sharedStoreWithName:kDefaultSmartStoreName];
+    SFSmartStore *store = [SFSmartStore sharedGlobalStoreWithName:kDefaultSmartStoreName];
```

Comment:
We are only getting a handle to the store to get the compile options.
Since we might not be logged in yet, we should get a global store rather than a user store.